src/ray/raylet/scheduling_queue.cc

```diff
@@ -0,0 +1,101 @@
+#ifndef SCHEDULING_QUEUE_CC
+#define SCHEDULING_QUEUE_CC
+
+#include "scheduling_queue.h"
+
+#include <list>
+
+#include "ray/status.h"
+#include "ray/id.h"
+
+namespace ray {
+
+const std::list<Task> &SchedulingQueue::waiting_tasks() const {
+  return this->waiting_tasks_;
+}
+
+const std::list<Task> &SchedulingQueue::ready_tasks() const {
+  return this->ready_tasks_;
+}
+
+const std::list<Task> &SchedulingQueue::scheduled_tasks() const {
+  return this->scheduled_tasks_;
+}
+
+const std::list<Task> &SchedulingQueue::running_tasks() const {
+  return this->running_tasks_;
+}
+
+const std::list<Task>& SchedulingQueue::ready_methods() const {
+  throw std::runtime_error("Method not implemented");
+}
+
+// Helper function to remove tasks in the given set of task_ids from a
+// queue, and append them to the given vector removed_tasks.
+void removeTasksFromQueue(
```

Comment:
PascalCase everywhere?  I've not seen any convention where local helpers has different casing than other functions.